INSTALL_mac.md

```diff
@@ -0,0 +1,78 @@
+# Instructions for macOS (10.9 or higher)
+
+JackTrip: A System for High-Quality Audio Network Performance over the Internet
+
+## Installation
+
+### Step 1: Install JACK
+
+Download and install the JACK2 OSX binary (0.92_b3) at <https://jackaudio.org/downloads/>. In addition to installing Jack in System directories, the installer will create a directory in Applications with the apps JackPilot and Qjackctl.
+
+### Step 2: Install JackTrip
+
+A binary for macOS is provided with each release. For the instructions below it is assumed that you will place the folder "jacktrip" in /Applications.
+
+### Step 3: Set permissions on JackTrip binary
+
+In Terminal, change directory to /jacktrip/bin/ as follows:
+
+cd /Applications/jacktrip/bin/
+
+Set the permissions for the jacktrip binary there:
+
+sudo chmod 755 jacktrip
+
+Enter your password when requested (it will not display). You can now use JackTrip from the /jacktrip/bin directory. 
```

Comment:
should this be `/Applications/jacktrip/bin`?

Reply:
Yes, good idea to include the full path to avoid confusion. Thanks. I'll make the udpate.